examples/report.rb

```diff
@@ -0,0 +1,36 @@
+require 'sogou/search/api/auth'
+require 'sogou/search/api/service/report'
+
+# ENV['SOGOU_API_TOKEN'] = 'xxxx'
```

Comment:
do we really need to keep these comments?

Reply:
yes, Perhaps, I will add more comment for them